perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -547,6 +542,40 @@ sub waste_munge_request_form_data {
     }
 }
 
+sub available_permissions {
```

Comment:
I don't think we need this function, we haven't been asked to have a special permission for this.